src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeRetrieverTest.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.EnvelopeAccessProperties;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.EnvelopeAccessProperties.Mapping;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.EnvelopeRepository;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Status;
+
+import java.util.List;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static uk.gov.hmcts.reform.bulkscanprocessor.helper.EnvelopeCreator.envelope;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class EnvelopeRetrieverTest {
+
+    @Autowired private EnvelopeRepository envelopeRepo;
+    @Mock EnvelopeAccessProperties accessProperties;
```

Comment:
can we not mock the properties - it's 100% configurable

Reply:
I personally prefer the mock solution in this case. It's easier to read. Also functional tests do the job without mocks already. Moreover removing the mocks in this case seems to me would only be testing that Spring is capable of loading the properties from the config file, which is really outside the scope of the test.